libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -1339,6 +1339,36 @@ bool AP_BLHeli::get_telem_data(uint8_t esc_index, struct telem_data &td)
     return true;
 }
 
+// get an individual ESC's temperature in degrees if available, returns true on success
+bool AP_BLHeli::get_temperature(uint8_t esc_index, int16_t& temp) const
+{
+    if (esc_index >= max_motors || last_telem[esc_index].timestamp_ms == 0) {
```

Comment:
rename timestamp_ms to last_update_ms

Reply:
This is scope creep :) I'd rather not start playing with the structure of the existing code